packages/macros/src/ember-addon-main.ts

```diff
@@ -70,6 +72,30 @@ export = {
       let appInstance = this._findHost();
       let source = appOrAddonInstance.root || appOrAddonInstance.project.root;
       babelPlugins.unshift(MacrosConfig.for(appInstance).babelPluginConfig(source));
+
+      let yarnLockPath = join(appInstance.project.root, 'yarn.lock');
```

Comment:
is it always guaranteed that `yarn.lock` exists at `project.root`? (eg, if a repo is using workspaces & the app/addon exists within a workspace)

Reply:
You are correct, `yarn.lock` might not exist in `project.root` but I'm not sure what else we can do in that scenario. 🤔

Reply:
Yeah, we could do a find up for `yarn.lock`/`package-lock.json`, but continue to fall back to `package.json` at `project.root`